config/database.yml

```diff
@@ -1,16 +1,18 @@
-development:
+default: &default
   adapter: postgresql
-  database: growstuff_dev
-  host: localhost
+  encoding: unicode
+  host: db
```

Comment:
Given that Docker for Windows and Mac is still in beta, though, we'll need something workable in the meantime.